examples/configs/pose_graph/pose_graph_synthetic.yaml

```diff
@@ -4,8 +4,8 @@ profile: True
 savemat: True
 
 num_poses: 256
-rotation_noise: 0.05
-translation_noise: 0.02
+rotation_noise: 0.02
+translation_noise: 0.05
```

Comment:
Hi @suddhu, we discussed this offline and converged on what's the correct version. Here's the summary and todo.

1. Convention to follow for this entire example: translation then rotation. Add a comment at the start of the example to make this clear.
2. In the yaml config the original numbers were correct (we can flip their order for consistency): translation_noise: 0.02, rotation_noise: 0.05
3. Your fixes so far corrected the order in three places. We missed one in line 276 since it does not use any identifiable params it was ambiguous. So we can flip the order there and add in-line comment for which row is trans/rot.
4. Update the output numbers for unit tests to pass with the above changes added.